lib/game.rb

```diff
@@ -0,0 +1,9 @@
+class Start
```

Comment:
It seems like maybe you started combining your intentions for this class with the runner. I'd pull most of the code from the runner into this class. Whatever your class name is should match the file name. So if you call this class `Start` you should name the file `start.rb`. I personally would probably keep the file name as `game.rb` and change the class name to `Game` because that more accurately describes the purpose of the class.